concent_api/verifier/tasks.py

```diff
@@ -179,6 +183,33 @@ def blender_verification_order(
     for file_path in source_files_list + [source_package_path]:
         delete_file(file_path)
 
+    blender_output_file_name = generate_blender_output_file_name(scene_file)
+    upload_file_name = generate_upload_file_name(subtask_id)
+    (file_size, file_check_sum) = get_file_size_and_checksum(blender_output_file_name)
+
+    try:
+        # Generate a FileTransferToken valid for an upload of the image generated by blender.
+        upload_file_transfer_token = create_file_transfer_token_for_concent(
```

Comment:
This should be outside of the `try ... except` block. An exception handler should cover only the code where you expect the exception.

Reply:
:+1: